idb/postgres/internal/writer/writer.go

```diff
@@ -246,6 +247,14 @@ func writeAssetResource(round basics.Round, resource *ledgercore.AssetResourceRe
 	}
 }
 
+func readSigmaDAOApp() (string, error) {
+	content, err := os.ReadFile("SigmaDAOApp.txt")
+	if err != nil {
+		log.Fatal(err)
+	}
+	return string(content), err
```

Comment:
why returning error? it is always going to be nil, since if it is not nil then we panic in `log.Fatal`.

Reply:
Ahh, yes